### DISCUSSION TITLE
problems after maui-check -dev

### DISCUSSION THREAD

**Discussion by benhakan** [dotnet/maui#1080] [Q&A]
Hi, some days ago I used 
maui-check -dev command.
This installed version .net 6.0.100-preview.4.21255.9
Later I removed preview 4.
Now maui-check shows no error. Everything is OK. But when I try to create a new or build a previous maui project with "dotnet new ..." or
"dotnet build ...."    this error occurs:  
Found .NET SDK, but did not find dotnet.dll at [C:\Program Files\dotnet\sdk\6.0.100-preview.4.21255.9\dotnet.dll]
I removed preview 4  and maui-check shows OK for preview 3. (I did restart my PC )
How to resolve this?
Thanks

**Comment by davidortinau** [comment]
@benhakan the tool was updated this morning for the public release of Preview 4. I recommend opening an issue on https://github.com/Redth/dotnet-maui-check and @Redth will look into it.
